.gitignore

```diff
@@ -0,0 +1,4 @@
+_site/
```

Comment:
you'll have to remove and re-add everything to apply the gitignore

[see this guide](https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-files)